release.py

```diff
@@ -46,10 +46,16 @@ def workspace_is_clean():
                          stderr=subprocess.STDOUT)
     return len(p.stdout.readlines()) == 0
 
-def is_draft_release(version_tag):
-    g = Github()
-    release = g.get_repo("buildbuddy-io/buildbuddy").get_release(version_tag)
-    return release.draft
+def is_draft_release(github_token, version_tag):
+    query_url = f"https://api.github.com/repos/buildbuddy-io/buildbuddy/releases/tags/{version_tag}"
```

Comment:
hm I'm not sure this `/tags/{tag}` endpoint works until the release is published - over in over in `buildbuddy-io/buildbuddy-ci-playground` I created a draft release v0.0.0 and then a published release v0.0.1, and if I curl this URL for v0.0.0 I get a 404 response even when authenticated, but the v0.0.1 release returns just fine.

Instead I think we could curl the latest page of `/releases` then scan for a release with the tag_name field == version_tag, and draft == True

Reply:
Ah good catch - updated